source/includes/_blueprints.md

```diff
@@ -144,6 +146,12 @@ Property | Type | Description
 -------- | ---- | -----------
 `application_file_name` | string | The main blueprint file name in the blueprint's archive.
 `blueprint_archive_url` | string | A URL the blueprint to be uploaded should be downloaded from by the manager.
+`availability` | string | Optional parameter, defines who can access the blueprint (default: tenant).
+
+Valid availability values are:
+* `private` - The resource is available to the owner (the creating user).
```

Comment:
I recommend -> `private` - The resource is available to the user that created the resource.